### ISSUE TITLE
Chrome 128 - window-size is ignored (chrome-for-testing, headless, linux)

### ISSUE THREAD

**Issue by yblatti** [symfony/panther#638]
I have a Symfony app (7.1).
For e2e tests I use [phpunit](https://symfony.com/doc/current/components/phpunit_bridge.html) + [symfony/panther](https://github.com/symfony/panther) + [Chrome for Testing](https://googlechromelabs.github.io/chrome-for-testing/) headless (on Debian Linux bullseye).

Since Chrome 128 has been released, my CI fails as the `--window-size` chrome argument is ignored.

I wrote a reproducer here : https://github.com/yblatti/repro-panther-screen-size-chrome

It seems to indicates that this has been introduced with Chrome 128.0.6575.0.

I have tried to have a look at https://chromium.googlesource.com/chromium/src/+log/128.0.6542.0..128.0.6543.0?pretty=fuller&n=10000
... but this way out of my league.

**Comment by yblatti** [comment]
I found that : https://chromium.googlesource.com/chromium/src/+/b9b39a430f71c710d16aafcc67278ef77440c18d

It seems that headless mode is changing, and we can use `--headless=old` to keep the previous behavior (for how long tho..)

**Comment by dunglas** [comment]
According to their docs, the `--window-size` flag is supported by both the old and the new mode: https://developer.chrome.com/docs/chromium/new-headless

It's maybe a bug in Chrome?

**Comment by yblatti** [comment]
Thanks for your time and feedback @dunglas :)

I reported it here : https://issues.chromium.org/issues/362522328

**Comment by Chris53897** [comment]
This worked for me. Just in case, someone is looking for a quick solution. I am not sure if the bug is solved.

```
$browser = Client::createChromeClient(
            arguments: [
                '--window-size=1400,800',
                '--headless=new',
            ],
 );
```

**Comment by llupa** [comment]
I had the same exact issue as OP with Chrome v130. The only solution for me was to use `--headless=old`

**Comment by codegain** [comment]
Had problem with setting window-size and setting the download directory via `download.default_directory`/`savefile.default_directory` after upgrading the chrome version to v131.

Using

```
PANTHER_NO_HEADLESS=true
PANTHER_CHROME_ARGUMENTS="--headless=old --disable-gpu --window-size=1920,1080"
```

instead of

```
PANTHER_CHROME_ARGUMENTS="--window-size=1920,1080"
```

fixed it for me.

**Comment by yblatti** [comment]
> PANTHER_CHROME_ARGUMENTS="--headless=old --disable-gpu --window-size=1920,1080"

Yes, but this is short term as chrome devs say _"We're happy to announce that the old Headless mode will be removed in Chrome 132"_ (source : https://developer.chrome.com/blog/removing-headless-old-from-chrome).

For now, I've switched to chrome-headless-shell (https://developer.chrome.com/blog/chrome-headless-shell).
